store/sqlstore/role_supplier.go

```diff
@@ -91,7 +100,7 @@ func (s *SqlSupplier) RoleSave(ctx context.Context, role *model.Role, hints ...s
 			return nil, model.NewAppError("SqlRoleStore.RoleSave", "store.sql_role.save.open_transaction.app_error", nil, err.Error(), http.StatusInternalServerError)
 		}
 		defer finalizeTransaction(transaction)
-		createdRole, appErr := s.createRole(ctx, role, transaction, hints...)
+		createdRole, appErr := s.createRole(role, transaction)
```

Comment:
why no hints?

Reply:
We are here simplifying the approach, the hints weren't used at all, and I followed the "You won't need it" approach here. I we finally need "hints" in certain calls, we can add explicitly there.